var/spack/repos/builtin/packages/py-rapidjson/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+from os import symlink
+from shutil import rmtree
+
+
+class PyRapidjson(PythonPackage):
```

Comment:
```suggestion
class PyPythonRapidjson(PythonPackage):
```
You'll also have to change the filename.

Reply:
Fixed!